### PR TITLE
Migrate service catalog and service management views to new translatable labels

### LINKED ISSUE #659: Migrate service catalog and service management views to new translatable labels
Migrate service catalog and service management views to new translatable labels.
Each resource has different statuses. Make sure we consolidate them into something more unified.

**Acceptance Criteria**

- [x] Follow Project DOD
- [x] extract common statuses and tooltips to common section on en.yaml
- [x] Migrate service management:
  - [x] catalog
  - [x] instances
  - [x] brokers
- [x] Migrate service catalog:
  - [x] catalog
  - [x] instances
  - [x] brokers

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Changes proposed in this pull request:

- Catalog - took (most of) them from [^2].
- Brokers - they can be either in `Ready` or `Failed` state[^1].
- BTP stuff - no enum for `reason`, it's just a string.

**Related issue(s)**

Closes #659 

[^2]: https://github.com/kubernetes-sigs/service-catalog/blob/master/pkg/controller/controller_instance.go
[^1]: https://github.com/kubernetes-sigs/service-catalog/blob/master/pkg/apis/servicecatalog/v1beta1/types.go#L353

### CODE REVIEW

**Review by dariadomagala — APPROVED** [review]

File: core/src/i18n/en.yaml [hunk 1]
```diff
@@ -964,6 +965,16 @@ instances:
   description: <0>Service Instance</0> is a provisioned instance of a service that can be consumed by an application.
   no-specific-plan-parameters:
     tooltip: The service provider did not define specific parameters for the selected plan. Refer to the documentation to learn about the required parameters, and define them as a JSON in the editor.
+  statuses:
```
Comment (grego952) [inline core/src/i18n/en.yaml:968]: 968-977: Do we need the uppercase for all words? Maybe we can use it only for the first one?

Comment (Wawrzyn321) [reply core/src/i18n/en.yaml:968]: We use the same convention in 293-295 & 801, so I decided to have it like this. They are to be uppercased anyway...

**Comment by Wawrzyn321** [comment]
Pod in `create-deployment.spec` was still `CONTAINER CREATING` - should we increase the timeout?
/test pre-busola-smoke-test-docker

File: core/src/i18n/en.yaml [hunk 2]
```diff
@@ -964,6 +965,16 @@ instances:
...
+    deprovisionedsuccessfully: Deprovisioned Successfully
+    deprovisionrequestunflight: Deprovision Request In Flight
```
Comment (alexandra-simeonova) [inline core/src/i18n/en.yaml:970]: In line 970 and 976, the variable is `deprovisionrequest**un**flight` but the label is `Deprovision Request In Flight`, shouldn't they both be `inflight`? 

```suggestion
    deprovisionrequestinflight: Deprovision Request In Flight
```

File: core/src/i18n/en.yaml [hunk 3]
```diff
@@ -964,6 +965,16 @@ instances:
...
+    instanceupdatedsuccessfully: Instance Updated Successfully
+    orphanmitigationsuccessful: Orphan Mitigation Successful
+    provisioncallfailed: Provision Call Failed
+    provisionedsuccessfully: Provisioned Successfully
+    provisioning: Provisioning
+    provisionrequestunflight: Provision Request In Flight
```
Comment (alexandra-simeonova) [inline core/src/i18n/en.yaml:976]:
```suggestion
    provisionrequestinflight: Provision Request In Flight
```

File: core/src/i18n/en.yaml (hunk 2 above)
Comment (Wawrzyn321) [reply core/src/i18n/en.yaml:970]: Damn!

**Review by alexandra-simeonova — APPROVED** [review]

**Review by dariadomagala — APPROVED** [review]

**Pull request merged**